src/i18n/strings/pl.json

```diff
@@ -1841,10 +1841,25 @@
         "files_button": "Pliki",
```

Comment:
These changes came from https://github.com/element-hq/matrix-react-sdk/commit/33c9c04598e7c6a813fab69d5d5078317f6d7175, I don't know whether I should remove them from my commit list to make the CI pass or just ignore it?